x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/find.ts

```diff
@@ -78,18 +78,41 @@ export default function createFindTests({ getService }: FtrProviderContext) {
         });
 
         it('should handle find alert request with filter appropriately', async () => {
+          const { body: createdAction } = await supertest
+            .post(`${getUrlPrefix(space.id)}/api/action`)
+            .set('kbn-xsrf', 'foo')
+            .send({
+              name: 'My action',
+              actionTypeId: 'test.noop',
+              config: {},
+              secrets: {},
+            })
+            .expect(200);
+
           const { body: createdAlert } = await supertest
             .post(`${getUrlPrefix(space.id)}/api/alert`)
             .set('kbn-xsrf', 'foo')
-            .send(getTestAlertData())
+            .send(
+              getTestAlertData({
+                enabled: false,
+                actions: [
+                  {
+                    id: createdAction.id,
+                    group: 'default',
+                    actionTypeId: 'test.noop',
+                    params: {},
+                  },
+                ],
+              })
+            )
             .expect(200);
           objectRemover.add(space.id, createdAlert.id, 'alert');
 
           const response = await supertestWithoutAuth
             .get(
               `${getUrlPrefix(
                 space.id
-              )}/api/alert/_find?filter=alert.attributes.alertTypeId:test.noop`
+              )}/api/alert/_find?filter=alert.attributes.actions:{ actionTypeId: test.noop }`
```

Comment:
Do we have previous examples of JSON being used on query string like this?
This seems non idiomatic, so just wondering what the reason is. 🤷‍♂️

Reply:
No examples I can think of, I added this here as we need a test to make sure we can filter on nested ES objects and this will become more common for the alerts list.

Reply:
This isn't JSON though, is it?  It's KQL AFAIK, which happens to _look_ like JSON for nested queries ...

Reply:
You're right, it's KQL and not JSON, looks very similar for nested queries.